pnpm-lock.yaml

```diff
@@ -1933,11 +1933,6 @@ importers:
 
 packages:
```

Comment:
it looks like all of the version changes below are downgrades - this is a bit weird isn't it? 🤔  probably not really important but I'm still wondering

Reply:
Yeah! Noticed that but am not fully sure why that's the case. @paescuj had also tested and he created the exact same lockfile too. 🤔

Reply:
We were wondering about the big diff as well, but it seems to be correct! I've regenerated the lockfile and it resulted into the same.

Reply:
alright 👍 should be fine then :)

Reply:
One explanation could be that other dependencies are depending on `jsonwebtoken` as well, resulting in quite some changes when we update to the same or a different version.

Reply:
actually, when I update `jsonwebtoken` on the main branch, the `pnpm-lock` file does not change as much - just three lines changed 🤔

Reply:
Thanks for checking, indeed seems to behave differently on `main`. I've now merge in `main` and regenerated the lockfile so it should be up-to-date.

Reply:
ah nice! now I can sleep better 😄 thanks! 🥳